README.md

```diff
@@ -1,10 +1,11 @@
 # Terraform Provider
 
+![Gitter](https://img.shields.io/gitter/room/DataDog/hashicorp-terraform)
+
 -   Website: https://www.terraform.io
--   [![Gitter chat](https://badges.gitter.im/hashicorp-terraform/Lobby.png)](https://gitter.im/hashicorp-terraform/Lobby)
```

Comment:
This badge was blurry and not well-positioned. I moved it up, and used `img.shields.io` which looks better.